rep-2006.rst

```diff
@@ -0,0 +1,73 @@
+REP: 2006
+Title: ROS 2 Vulnerability Disclosure Policy
+Author: ROS 2 Security Working Group, Chris Lalancette <clalancette@openrobotics.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 12-May-2020
+Post-History:
+
+
+Abstract
+========
+
+This REP describes the Vulnerability Disclosure Policy related to the ROS 2 Common Packages as defined in `REP-2005 <https://www.ros.org/reps/rep-2005.html>`_.
+
+
+Motivation
+==========
+
+The developers of the Robot Operating System (ROS) take security very seriously.
+As such, we would like to be informed when a security bug is found so that it can be fixed and disclosed as quickly as possible.
+The rest of the document outlines what is covered by this policy and how to report security vulnerabilities.
+
+Scope
+=====
+
+The Vulnerability Disclosure Program outlined here covers all code within the ROS 2 Common Packages as defined in REP 2005.
+
+The ROS Security Working Group may also be able to assist you with reporting vulnerabilities which are not in scope of this policy but which are related to ROS or more generally to robotics.
+An increasing number of companies are bringing ROS-based products to market, and during this growth period we anticipate vulnerabilities to be identified in products before vendors have an established vulnerability reporting program.
+We will work with you on a best-effort basis to help connect you with responsible parties best suited to address your concerns.
+
+How to submit a vulnerability
+=============================
+
+Please submit vulnerability reports by emailing security@openrobotics.org, preferably in English.
+This is a private list of security officers who can help verify the bug report and develop and release a fix.
+The more information provided about the bug, the easier it will be to fix.
+If you already have a fix, please include it with your report.
+We may ask to exchange PGP keys to discuss sensitive details about the vulnerability which may include proof-of-concept code, an impact assessment, recommended remediation steps or other information that will help us find and fix the problem.
+
+Include any plans you may have to disclose details about the vulnerability.
+In order to protect our users, unless otherwise agreed by both parties, we ask that you not publicly discuss the vulnerability until at least 90 days have passed or the fix is published.
```

Comment:
> 90 days have passed

since? The first mail I assume? I think it should be explicit anyways.

Reply:
Yeah, good point.  I'll make that explicit.